docker-compose.yaml

```diff
@@ -0,0 +1,9 @@
+version: '3.8'
+services:
+  gamestonkterminal:
+    build:
+      context: .
+    image: gamestonkterminal:dev
+    dns:
+      - 1.1.1.1
+      - 8.8.8.8
```

Comment:
Curious, why the DNS server specification?

Reply:
Sure, I'll remove it.

Reply:
Oh no, I'm just curious about your reasoning 😄 I'm not the maintainer, I'm just trying to help out and possibly learn something along the way.